lib/config/user.js

```diff
@@ -61,7 +85,102 @@ class AuthProvider {
       this.url = url;
     }
     this.idp_certificate_authority = ca;
-  }
+  }  
+}
+
+/**
```

Comment:
It seems there are too much comments block there 😉!

Reply:
Done :)

Reply:
Ah I was just thinking about the extra `//` or `/*`. Otherwise, feel free to add the comments if you think these are important.